app/styles/layout.scss

```diff
@@ -0,0 +1,125 @@
+.fullpage {
+  display: flex;
```

Comment:
Make this element also `height: 100vh` and this will help make separate scrolling areas.